app/cdap/components/ConnectionList/index.tsx

```diff
@@ -49,13 +54,14 @@ export default function ConnectionList() {
   const { connectorType } = useParams() as IRecords;
 
   const refs = useRef([]);
+  const headersRefs = useRef([]);
   const classes = useStyles();
   const loc = useLocation();
   const queryParams = new URLSearchParams(loc.search);
   const pathFromUrl = queryParams.get('path') || '/';
   const [loading, setLoading] = useState(true);
   const [isErrorOnNoWorkspace, setIsErrorOnNoWorkSpace] = useState<boolean>(false);
-
+  const [tabSize, setTabSize] = useState<number>(0);
```

Comment:
what is this tabSize, is it the width of the tab or the length of the list
plz check and rename as appropriate

Reply:
Done